javascript/atoms/BUILD.bazel

```diff
@@ -250,11 +250,12 @@ closure_js_library(
     ],
     deps = [
         ":bot",
+        ":css",
         ":dom",
         ":errors",
         ":json",
         ":useragent",
-        ":xpath",
+        "//third_party/js/wgxpath",
```

Comment:
So, do we need to depend on wicked good xpath here?

Reply:
Yes, this is still needed here for legacy browsers to have XPath locator support.

This PR does only remove the need for wicked good xpath for is-displayed atom.

Reply:
Which legacy browsers require it? IE 11?